packages/fxa-payments-server/src/routes/Product/index.test.tsx

```diff
@@ -94,23 +94,31 @@ describe('routes/Product', () => {
   const initApiMocks = (displayName?: string) => [
     nock(profileServer)
       .get('/v1/profile')
-      .reply(200, { ...MOCK_PROFILE, displayName }),
+      .reply(
+        200,
+        { ...MOCK_PROFILE, displayName },
+        { 'Access-Control-Allow-Origin': '*' }
```

Comment:
Might be good at some point to extract this out into a constant to DRY things up